Files/ViewModels/AdaptiveSidebarViewModel.cs

```diff
@@ -21,7 +21,14 @@ namespace Files.ViewModels
     public class AdaptiveSidebarViewModel : ObservableObject, IDisposable
     {
         public ICommand EmptyRecycleBinCommand { get; private set; }
-        public IPaneHolder PaneHolder { get; set; }
+
+        private IPaneHolder paneHolder;
+        public IPaneHolder PaneHolder
+        {
+            get => paneHolder;
+            set => SetProperty(ref paneHolder, value);
+        }
```

Comment:
In the future, let's open a new PR for unrelated changes.

Reply:
@yaichenbaum well it isn't unrelated, I added it trying to get the oneway bindings to work but forgot to take it out :). Unneeded though, yes.

Reply:
Hmm it looks like this may have been needed after all. Without it the binding to show the info text never updates and never shows.